.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.3'
+          php-version: '7.4'
```

Comment:
Maybe start with PHP 8.0 as min?

Reply:
I would like to leave that. because of SW5 I often use PHP 7.4

but it annoyed me, that I could not use the tools, if I am on PHP 8. 

so leaving out PHP 7.4 will not improve my situation :smile: